test/indexes.test.js

```diff
@@ -120,107 +120,126 @@ describe('Indexes', function () {
       assert.deepEqual(idx.tree.search('world'), []);
       assert.deepEqual(idx.tree.search('bloup'), []);
     });
-	
```

Comment:
This file diff looks bad but it's really just one new test (lines 123-141), and then properly indenting the "Array fields" suite/description.